### PR TITLE
26905c37 Ft login 52

### PR DESCRIPTION
**Pull request by Evaristekanova**
### how does this PR do?

- adding login functionalities
- fetching data from API

### how this should be tested?
- do `npm i` or `npm install`
- run `npm ru dev` in your terminal

### screenshoots

- how the design looks on large screen
Screenshot (14)

- responsive
Screenshot (12)

- how it looks when login succeed
Screenshot (15)

### CODE REVIEW

**Comment by Evaristekanova** [comment]
[Users should be able to Login with Email and Password](https://trello.com/c/MWm3PPyN/52-users-should-be-able-to-login-with-email-and-password)

**Comment by Evaristekanova** [comment]
[Users should be able to Login with Email and Password](https://trello.com/c/MWm3PPyN/52-users-should-be-able-to-login-with-email-and-password)

File: .eslintrc.cjs [hunk 1]
```diff
@@ -4,11 +4,9 @@ module.exports = {
     es2021: true,
   },
   extends: [
-    'airbnb',
```
Comment (Aimeana100) [inline .eslintrc.cjs]: eeeh, why are you editing Eslint configurations ?? 😢

**Review by Christabll — APPROVED** [review]

**Review by Aimeana100 — APPROVED** [review]
Works perfect

**Review by ngoma3 — APPROVED** [review]

File: src/assets/GOOGLE.png
Comment (mansurissa) [inline src/assets/GOOGLE.png:1]: Yesterday we suggested to use a white google icon.

**Review by mansurissa — APPROVED** [review]

**Pull request merged**